### PR TITLE
52 upgrade clap 2

### PR DESCRIPTION
**Pull request by levBagryansky**
Solving #52 
Upgade clap version to `4.0.14`

### CODE REVIEW

**Comment by levBagryansky** [comment]
@yegor256 please take a look

**Comment by levBagryansky** [comment]
@yegor256 please check

File: src/bin/reo.rs [hunk 1]
```diff
@@ -263,6 +258,7 @@ pub fn main() -> Result<()> {
                     start.elapsed()
                 );
             }
+            print!("HELLO");
```
Comment (yegor256) [inline src/bin/reo.rs:261]: @levBagryansky typo?

**Comment by yegor256** [comment]
@levBagryansky one typo

**Comment by levBagryansky** [comment]
@yegor256 please check

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**